bundles/org.eclipse.equinox.p2.core/META-INF/MANIFEST.MF

```diff
@@ -74,7 +74,7 @@ Service-Component:
 Import-Package: org.bouncycastle.bcpg;version="1.65.0",
  org.bouncycastle.openpgp;version="1.65.0",
  org.eclipse.core.runtime;common=split;version="[3.5.0,4.0.0)",
- org.eclipse.equinox.p2.core;version="[2.8.0,2.9.0)";resolution:=optional,
+ org.eclipse.equinox.p2.core;version="[2.12.0,2.13.0]";resolution:=optional,
```

Comment:
`org.eclipse.equinox.p2.core;version="[2.12.0,2.13.0]"` ???

Why do we import same package we provide? Is this something that can be provided by others???

Reply:
> Why do we import same package we provide? Is this something that can be provided by others???

Importing exported packages is considered good OSGi practice by many:
https://felix.apache.org/documentation/tutorials-examples-and-presentations/apache-felix-osgi-faq.html#import-own-exports
I have not yet seen the necessity of that for Eclipse, but I have also not thought about it in great depth.